service/provider/src/main/java/gov/va/vro/service/provider/camel/PrimaryRoutes.java

```diff
@@ -64,9 +68,29 @@ private void configureRouteClaimSubmitForFull() {
         .process(FunctionProcessor.fromFunction(saveToDbService::insertClaim))
         // Use Properties not Headers
         // https://examples.javacodegeeks.com/apache-camel-headers-vs-properties-example/
+        .setProperty("id", simple("${body.recordId}"))
```

Comment:
This surprises me. I thought these audit saves were not supposed to change the messages. Looks like they do. I do not really know the implication. I would have preferred these did not changed the message and this property is set within the processor.  The downstream microservice certainly does not expect these additional fields that are artifacts of the database record.